tools/truncate-silence.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash -e
+tools_path=$(dirname "$0")
+
+usage() {
+    echo "Usage: $0 [OPTION] [FILE]" 1>&2;
+    echo "Truncates silence from the start or the end of an audio file"
+    echo "    -c    Check ffmpeg." 1>&2;
```

Comment:
Any reason not to check ffmpeg unconditionally?

Reply:
Because I wouldn't like to limit it only for webm files. Since check_ffmpeg fails if webm codecs are not available.